Framework/Kernel/inc/MantidKernel/ContainerDtype.h

```diff
@@ -0,0 +1,80 @@
+#ifndef MANTID_KERNEL_CONTAINERDTYPE_H_
+#define MANTID_KERNEL_CONTAINERDTYPE_H_
+
+#include <string>
+#include <type_traits>
+#include <vector>
+
+#include <boost/python/class.hpp>
```

Comment:
Our current structure limits boost::python interaction to a single project: `PythonInterface`. As this is file is only used by that library can we move it to `PythonInterface/inc/MantidPythonInterface/kernel/Converters`? `Kernel` has gotten very heavy because things have tended to be put there that we think we might use rather than knowing we will.

It also looks to me that none of these headers are actually used anyway...?